tests/test_sqlserver.py

```diff
@@ -0,0 +1,11 @@
+def test_connection(db_connection):
+    try:
+        cursor = db_connection.connection.cursor()
+
+        for row in cursor.execute("SELECT * from ShoppingList;"):
```

Comment:
What would be returned when I run the tests for the first time? Who is creating the ShoppingList on step 3?

Reply:
You're right. In Step 3 I assume the databases TestDB and ShoppingDB are already made and they have the right tables (TestDB tables have some values). I would rather fix this in Step 4, if that's ok? I already have the init of databases within dockerfiles when building containers for app and database.